app/views/moderations/mod.html.erb

```diff
@@ -91,6 +91,11 @@
   input.undo {
     background: #ff0000; /* $red */
   }
+
+  .adjustment-error {
```

Comment:
Probably for another PR, but we should start getting into the habit of putting these into SASS files and referencing the generated CSS file in backend templates. This way the styles are cached in the browser instead of being served up every time this template gets loaded.